core/services/ocr2/plugins/llo/integration_test.go

```diff
@@ -342,11 +342,6 @@ func TestIntegration_LLO(t *testing.T) {
 	multiplier := decimal.New(1, 18)
 	expirationWindow := time.Hour / time.Second
 
-	reqs := make(chan request, 100000)
-	serverKey := csakey.MustNewV2XXXTestingOnly(big.NewInt(-1))
-	serverPubKey := serverKey.PublicKey
```

Comment:
the t.Run(...) tests do not call t.Parallel() , so I am not sure how moving this to individual tests would help.

Would it make sense to keep this PR only about disabling tests and not modifying them?

Reply:
Thanks, this makes total sense. In this case, would it be possible to put this stanza inside of startMercuryServer instead of duplicating it?